src/main/java/au/org/emii/ncdfgenerator/CreateWritable.java

```diff
@@ -2,29 +2,44 @@
 package au.org.emii.ncdfgenerator;
```

Comment:
I chose ncdfgenerator to avoid confusion with other netcdf-generator implementations. I'm open to further arguments on relying on namespacing.